File: versioned_docs/version-1.20.x/examples/js/src/lifecycle_policy_mode.mjs
import { Client, LifecycleMode } from "reduct-js";

const client = new Client("http://127.0.0.1:8383", { apiToken: "my-token" });

await client.setLifecycleMode("my-lifecycle", LifecycleMode.DRY_RUN);
await client.setLifecycleMode("my-lifecycle", LifecycleMode.DISABLED);
await client.setLifecycleMode("my-lifecycle", LifecycleMode.ENABLED);

File: versioned_docs/version-1.20.x/examples/py/src/lifecycle_policy_mode.py
import asyncio

from reduct import Client, LifecycleMode


async def main():
    async with Client("http://127.0.0.1:8383", api_token="my-token") as client:
        await client.set_lifecycle_mode("my-lifecycle", LifecycleMode.DRY_RUN)
        await client.set_lifecycle_mode("my-lifecycle", LifecycleMode.DISABLED)
        await client.set_lifecycle_mode("my-lifecycle", LifecycleMode.ENABLED)


asyncio.run(main())
